--- qcomicbook-0.4.0/fedora/qcomicbook.desktop	2007-11-05 22:16:28.000000000 +0100
+++ qcomicbook.desktop	2008-03-08 22:26:03.000000000 +0100
@@ -7,10 +7,10 @@
 Comment=QComicBook is a viewer for comic book archives (cbz, cbr, cbt). QComicBook has specific features like page preview, thumbnails view, bookmarks, manga mode for easing comicbook reading.
 Comment[tr]=QComicBook cbz, cbr, cbt biçimindeki çizgi roman arşivlerini görüntülemek için kullanılır. Sayfa ön görünümü, sayfa döşeme, yer imi, mangalar için sayfa sırasını ters gösterme gibi çizgi roman okumayı kolaylaştırıcı özellikleri bulunuyor.
 Comment[pl]=Przeglądarka do archiwów z komiksami
-Exec=qcomicbook
-Icon=/usr/share/qcomicbook/qcomicbook.png
+Exec=qcomicbook  %f
+Icon=qcomicbook
 Terminal=false
 Type=Application
 StartupNotify=false
-Categories=Application;Graphics;Viewer;
+Categories=Qt;Graphics;Viewer;
 MimeType=application/x-cbz;application/x-cbr;application/x-cbt;
